src/graph/graph_context.c

```diff
@@ -0,0 +1,23 @@
+#include "graph_context.h"
```

Comment:
See PR #134 for a somewhat more well-considered version of this code.

What's present in this PR now is functional, save that the context members are not persisted, so index retrievals fail once the server has been restarted.